core/domain/blog_services.py

```diff
@@ -199,6 +209,30 @@ def get_blog_post_summary_by_id(
         return None
 
 
+def get_blog_post_summary_models_by_ids(
+    blog_post_ids: List[str]
+) -> List[blog_domain.BlogPostSummary]:
+    """Given the list of blog post IDs, it returns the list of blog post summary
+    domain object.
+
+    Args:
+        blog_post_ids: List[str]. The list of blog post IDs for which blog post
+            summaries are to be fetched.
+
+    Returns:
+        List[BlogPostSummary]. The list of blog post summary domain object
+        corresponding to the given list of blog post IDs.
+    """
+    blog_post_summary_models = blog_models.BlogPostSummaryModel.get_multi(
+        blog_post_ids)
+    blog_post_summaries = []
+    blog_post_summaries = [
+        get_blog_post_summary_from_model(model)
+        for model in blog_post_summary_models if model is not None
+    ]
+    return (blog_post_summaries if len(blog_post_summaries) != 0 else [])
```

Comment:
This is unneeded, if en(blog_post_summaries) == 0 then the `blog_post_summaries` will be `[]` anyway.

Reply:
Done

Reply:
Done

Reply:
Done